typings/index.d.ts

```diff
@@ -1,95 +1,92 @@
-// Type definitions for CodeceptJS 1.0.2
+// Type definitions for CodeceptJS
 // Project: https://github.com/codeception/codeceptjs/
-// Definitions by: Michael Bodnarchuk <http://github.com/DavertMik>, Drew Diamantoukos <https://github.com/KennyRules>
-
-declare class Locator {
-  or(locator): Locator;
-  find(locator): Locator;
-  withChild(locator): Locator;
-  find(locator): Locator;
-  at(position): Locator;
-  first(): Locator;
-  last(): Locator;
-  inside(locator): Locator;
-  before(locator): Locator;
-  after(locator): Locator;
-  withText(text): Locator;
-  withAttr(attr): Locator;
-  as(output): Locator;
-}
+/// <reference path="./types.d.ts" />
+/// <reference path="../node_modules/webdriverio/webdriverio-core.d.ts" />
```

Comment:
webdriverio may not be in codeceptjs node_modules. But can be in the root of the node_modules folder. 
And webdriverio can be not installed